R/methods-Chromatograms.R

```diff
@@ -430,3 +430,15 @@ setMethod("mz", "Chromatograms", function(object) {
     .mz_chromatograms(object, mz = "mz")
 })
 
+#' @rdname Chromatograms-class
+#'
+#' @description
+#'
+#' \code{polarity}: returns the polarity of the scans/chromatograms: `1`,
+#' `0` or `-1` for positive, negative or unknown polarity.
```

Comment:
Is polarity really encoded as 1 for positiv, 0 for negative and -1 for unknown?
```
[Term]
id: MS:1000465
name: scan polarity
def: "Relative orientation of the electromagnetic field during the selection and detection of ions in the mass spectrometer." [PSI:MS]
relationship: part_of MS:1000441 ! scan
```
I have never seen anything else than a "positive scan" (and never touched any SRM/MRM data) but IHMO it would be logic to encode a "negative scan" as minus one (and unknown as zero or something different).

Reply:
I know - having +1 for positive, -1 for negative and NA for unknown would make more sense. I believe it comes from the `ramp` c++ library - and `mzR` uses that for all backends (`Ramp`, `pwiz`) - there has been an issue on that because I was wondering about that encoding too (can't find the issue - could be in `mzR` or `xcms`). 
The `header` function in `mzR` definitely returns this encoding and that's also the value we add to the `Spectrum` objects in `readMSData`.